msa/black-box-tests/src/test/java/org/thingsboard/server/msa/ui/base/AbstractDriverBaseTest.java

```diff
@@ -70,6 +71,15 @@ abstract public class AbstractDriverBaseTest extends AbstractContainerTest {
     private JavascriptExecutor js;
     public static final long WAIT_TIMEOUT = TimeUnit.SECONDS.toMillis(10);
     private final Duration duration = Duration.ofMillis(WAIT_TIMEOUT);
+    public final int MAX_FAILED_TESTS = 20;
+    public static int failedTestsCount = 0;
```

Comment:
I expect to use some TestNG framework functionality to achieve the goal. Similar to the JUnit runner.
The static variable approach might not work for parallel test runs with forked JMVs.

Thread unsafe.

Does not init value by the test framework. So multiple runs on the same machine will always skip if some test fails before.